src/github.com/stellar/gateway/bridge/handlers/request_handler_builder.go

```diff
@@ -40,9 +41,15 @@ func (rh *RequestHandler) Builder(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	sequenceNumber, err := strconv.ParseUint(request.SequenceNumber, 10, 64)
-	if err != nil {
-		errorResponse := protocols.NewInvalidParameterError("sequence_number", request.SequenceNumber, "Sequence number must be a number")
+	if request.SequenceNumber == "" {
```

Comment:
Yes, you need to check for errors even if the code is longer. It will be easier to debug later.

For the code in this comment:
```go
accountResponse, err := rh.Horizon.LoadAccount(request.Source)
if err != nil {
```
You aren't logging the error returned by `LoadAccount`.
```go
if err != nil {
			errorResponse := protocols.NewInvalidParameterError("sequence_number", accountResponse.SequenceNumber, "Could not parse sequence number from network")
			log.WithFields(errorResponse.LogData).Error(errorResponse.Error())
			server.Write(w, errorResponse)
			return
		}
```
This code is exactly the some except error message. Can you just leave a single one and just update the message after checking if the seqnum was loaded from the network or not?